engine/engine/src/engine.cpp

```diff
@@ -1818,12 +1820,12 @@ void dmEngineDestroy(dmEngine::HEngine engine)
 {
     engine->m_RunResult.Free();
 
+    Delete(engine);
```

Comment:
I prefer deleting things in reverse-order.
As it was, the webserver was deleted before the extension could remove the http handler.